config/xml_schemas/env_mach_specific.xsd

```diff
@@ -5,7 +5,7 @@
 <xs:attribute name="version" type="xs:decimal"/>
 <xs:attribute name="lang" type="xs:NCName"/>
 <xs:attribute name="compiler" type="xs:string"/>
-<xs:attribute name="debug" type="xs:boolean"/>
+<xs:attribute name="DEBUG" type="upperBoolean"/>
```

Comment:
@jedwards4b I don't know why this change was made (changing debug to DEBUG). Is that going to break you guys?

Reply:
@amametjanov , looks like you made this change. Care to chime in?

Reply:
Isn't this tied to the changes in env_mach_specific?  It will require some changes in the config_machines.xml file on the cesm side, but I think that it is a good change.